nixos/modules/services/misc/dendrite.nix

```diff
@@ -89,7 +103,7 @@ in
           };
           private_key = lib.mkOption {
             type = lib.types.path;
-            example = "${workingDir}/matrix_key.pem";
+            example = "/$CREDENTIALS_DIRECTORY/private_key";
```

Comment:
```suggestion
            example = "$CREDENTIALS_DIRECTORY/private_key";
```

Reply:
Yes, it's ugly. However, to pass the type check of `lib.types.path`, the prefix `/` is needed.

Reply:
Then make it a path or a string starting with `$CREDENTIALS_DIRECTORY`

Reply:
Done